jerry-core/jerry.c

```diff
@@ -620,24 +439,20 @@ jerry_release_object (jerry_object_t *object_p) /**< pointer acquired through je
  *
  * Warning:
  *         Acquired pointer should be released with jerry_release_value
- *
- * @return pointer that may be used outside of the engine
  */
-jerry_value_t *
-jerry_acquire_value (jerry_value_t *value_p) /**< API value */
+void
```

Comment:
This is incorrect. For strings it might return with a new instance. Although I don't like that idea, because the same string might be copied several times.